deploy/configs/externals/modules.yaml

```diff
@@ -40,6 +41,7 @@ modules:
       - emacs
       - environment-modules
       - ffmpeg
+      - flex@2.6.4
```

Comment:
Ideally, we do not need to pin package version in module.yaml.
Also, I see that `deploy/configs/packages.yaml` had external flex/bison. I have removed it in my PR. Lets see how it goes.

Reply:
#1191 looks good now, I'll close this.